python/remotivelabs-broker/remotivelabs/broker/sync/helper.py

```diff
@@ -86,17 +91,23 @@ def publish_signals(client_id, stub, signals_with_payload, frequency: int = 0):
     except grpc._channel._Rendezvous as err:
         print(err)
 
-def printer(signals):
+
+def printer(signals: Sequence[common_pb2.SignalId]) -> None:
```

Comment:
This is only used if the client program invokes it?

Reply:
Yes